doc/tutorial/ranges/index.md

```diff
@@ -125,7 +125,7 @@ std::views::reverse v{vec};
 That's because `std::views::reverse` is not the view itself, it's an *adaptor* that takes the underlying range
 (in our case the vector) and returns a view object over the vector.
 The exact type of this view is hidden behind the `auto` statement.
-This has the advantage, that we don't need to worry about the template arguments of the view type, but more importantly
+This has the advantage that we don't need to worry about the template arguments of the view type, but more importantly
 the adaptor has an additional feature: it can be *chained* with other adaptors!
```

Comment:
```suggestion
This has the advantage that we don't need to worry about the template arguments of the view type. 
The adaptor has another, very useful feature: it can be *chained* with other adaptors!
```

I had to read the sentence a few times to notice that the feature of chaining has nothing to do with the explanation of the advantage of `auto` :)